airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -294,6 +299,10 @@ def execute(self, context) -> Optional[str]:
                 self.log.info("creating pod with labels %s and launcher %s", labels, launcher)
                 final_state, _, result = self.create_new_pod_for_operator(labels, launcher)
             if final_state != State.SUCCESS:
+                if self.retry_only_on_pod_launching_failure:
```

Comment:
Is there any other way this condition could be hit?

I'm just not 100% sure on the name for this setting -- I can see it useful but (in my sleep deprived state) I'm struggling to parse the docs/name of the param to work out what it actually does.

Reply:
Maybe I have mistake in doc.
doc for this param says:
```
    :param retry_only_on_pod_launching_failure: retry logic is only effective if pod launching fails. This
        prevents retries when image failures occurs. Useful for non-idempotent tasks.
    :type retry_only_on_pod_launching_failure: bool
```
As far as I understand Idempotence (https://en.wikipedia.org/wiki/Idempotence) the result of idempotent task will be the same if we run it once again. So if it fails then it make no sence to retry, it will fail again.
Looks like we should change 
```
Useful for non-idempotent tasks.
```
with 
```
Useful for idempotent tasks.
```

Reply:
Fixed doc.
Let me know if you have better name for this parameter.